addon/components/step-manager/step.js

```diff
@@ -26,6 +27,18 @@ export default Component.extend({
     }
 
     this['register-step'](this);
+
+    // This is a hacky fix to make the addon work on Ember.js <= 2.8. The
+    // reason is that the VisibilitySupport mixin that is implemented in
+    // every component requires the component to be in the DOM. In particular,
+    // the line at https://github.com/emberjs/ember.js/blob/v2.8.0/packages/ember-views/lib/mixins/visibility_support.js#L46
+    // is failing.
+    //
+    // As the component is not rendered in the DOM, we don't need any of the
+    // functionality provided by VisibilitySupport. So the fix is "ok".
+    if (isFunction(this._toggleVisibility)) {
```

Comment:
Unfortunately it's needed for 2.8 as well :-( (see https://travis-ci.org/alexlafroscia/ember-steps/builds/261485132 for the Travis build right before the commit with the fix).